validator/client/beacon-api/beacon_api_validator_client.go

```diff
@@ -14,140 +14,252 @@ import (
 )
 
 type beaconApiValidatorClient struct {
-	url        string
-	httpClient http.Client
+	url            string
+	httpClient     http.Client
+	fallbackClient iface.ValidatorClient
 }
 
-func (*beaconApiValidatorClient) GetDuties(_ context.Context, _ *ethpb.DutiesRequest) (*ethpb.DutiesResponse, error) {
+func NewBeaconApiValidatorClient(url string, timeout time.Duration) iface.ValidatorClient {
```

Comment:
Keeping the "old" constructor for unit testing purposes, since in that situation we don't want fallbacks to be affecting the results.